api.py

```diff
@@ -13,6 +13,18 @@ def index():
     return 'Dino'
 
 
+@app.route('/personal-leaderboard', methods=['GET'])
+def get_personal_leaderboard():
+    username = request.args.get('user', type=str) 
+    data = select("""SELECT score, date, username FROM scores 
+    JOIN users ON  users.user_id = scores.user_id 
+    WHERE username =  %s""",(username,))
```

Comment:
two spaces between = and %s here as well

Reply:
how do you even see these hahaha impressive eye